src/modules/storage/LedgerStorage.ts

```diff
@@ -235,4 +236,44 @@ export class LedgerStorage extends Storages
             callback(err, rows);
         });
     }
+
+    /**
+     * Get validators
+     * @param height: The height parameter is optional.
+     * validators based on the block height if there is a height.
+     * if height null the most up to date state is expected.
+     * if the height is null then current valid validators.
+     * @param Callback If provided, this function will be called when
+     * the database was finished successfully or when an error occurred.
+     */
+    public getValidators (height: number | null, address: string | null,
+        callback: (err: Error | null, rows: any[]) => void)
+    {
+        var sql =
+        `SELECT tx_outputs.address,
+                enrollments.block_height as enrolled_at,
+                enrollments.utxo_key as stake,
+                (? - (enrollments.block_height + 1)) as distance,
+                enrollments.random_seed
+        FROM enrollments
+            LEFT JOIN tx_outputs ON enrollments.utxo_key = tx_outputs.utxo_key
+        WHERE 1=1
+            and enrollments.block_height >= (? - enrollments.cycle_length)
+            and enrollments.block_height <= ?
```

Comment:
Since you used uppercase for other SQL keywords, let's be consistent and use `AND` here.